cardform/src/main/java/com/mercadolibre/android/cardform/presentation/ui/custom/InputFormEditText.kt

```diff
@@ -207,7 +207,12 @@ internal class InputFormEditText(context: Context, attrs: AttributeSet?, defStyl
                     }
 
                     update(this, newText.substringBefore('$')) {
-                        setText(it)
+                        setText(
+                            if (it.takeLast(1).isBlank())
```

Comment:
@luissilva-meli me podrías dar más contexto de lo que sucede? Cual es el estado inconsistente de `it` para hacer este chequeo?

Reply:
@jorGonzalez291292 proporcionó una descripción más detallhada del error.